airflow/serialization/helpers.py

```diff
@@ -32,9 +32,10 @@ def serialize_template_field(template_field: Any) -> str | dict | list | int | f
     def is_jsonable(x):
         try:
             json.dumps(x)
-            return True
         except (TypeError, OverflowError):
             return False
+        else:
```

Comment:
Even the else is arguably not needed

Reply:
It isn't, but it aligns nicely with the `return False` partner.

Reply:
Agree. We have not used too much of the `try/except/else` but I started to like it, it's much more explicit than return false in this case.